src/index.ts

```diff
@@ -2,5 +2,5 @@ export * as profile from './profile';
 export * as session from './session';
 
 export { config } from './profile';
-export { getIdentity, getIdentities, deleteIdentity } from './identity';
-export { authorize, IEveryAuthOptions } from './authorize';
+export { getIdentity, getIdentities, deleteIdentity, deleteIdentities } from './identity';
+export { authorize, IEveryAuthOptions, getHostedBaseUrl } from './authorize';
```

Comment:
why export getHostedBaseUrl?

Reply:
for testing

Reply:
testing can do `import { getHostedBaseUrl } from '../src/authorize'` directly, though.  No need to export it from the module at large, is there?

Reply:
fixed